packages/generator/pyproject.toml

```diff
@@ -26,7 +26,7 @@ grpcio = "^1.49.1"
 protobuf = "^4.21"
 black = ">=24.8.0"
 click-option-group = ">=0.5.6"
-# ni-measurement-plugin-sdk-service = {version = "^2.0.0"}
+ni-measurement-plugin-sdk-service = {version = "^2.1.0-dev1", allow-prereleases = true}
```

Comment:
Do you think we should halt this PR because the resolve_service API signature was updated in #925?

Reply:
no